sdk/spring/compatibility-tests.yml

```diff
@@ -10,6 +10,10 @@ stages:
       - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
         parameters:
           SparseCheckout: true
+          SparseCheckoutPaths:
```

Comment:
You'll also need to add the python file referenced in the PreGenerationSteps below.

Reply:
Maybe just including `sdk/spring/scripts` is most flexible.